lib/test/Test/Pos/Launcher/ConfigurationSpec.hs

```diff
@@ -5,22 +5,48 @@ module Test.Pos.Launcher.ConfigurationSpec
 import           Universum
 
 import           Data.Time.Clock.POSIX (getPOSIXTime)
-import           Test.Hspec (Spec, describe, it, shouldSatisfy)
+import           Test.Hspec (Spec, describe, it, shouldBe, shouldSatisfy)
 
+import           Ntp.Client (NtpConfiguration)
+import qualified Pos.Chain.Genesis as Genesis (Config (..))
+import           Pos.Chain.Txp (TxpConfiguration)
 import           Pos.Core.Slotting (Timestamp (..))
+import           Pos.Crypto (ProtocolMagic (..), RequiresNetworkMagic (..))
 import           Pos.Launcher.Configuration (ConfigurationOptions (..),
-                     defaultConfigurationOptions, withConfigurations)
+                     WalletConfiguration, defaultConfigurationOptions,
+                     withConfigurations)
 import           Pos.Util.Config (ConfigurationException)
 import           Pos.Util.Wlog (setupTestLogging)
 
+configFilePath :: FilePath
+configFilePath = "configuration.yaml"
+
+checkYamlSection :: Text -> Spec
+checkYamlSection key = describe ("key: " ++ show key) $ do
+    it "should be RequiresNoMagic" $ do
+        liftIO $ setupTestLogging
+        startTime <- Timestamp . round . (* 1000000) <$> liftIO getPOSIXTime
+        let cfo = ConfigurationOptions configFilePath key (Just startTime) Nothing
+        rnm  <- liftIO (withConfigurations Nothing Nothing False cfo getRNM)
+        rnm `shouldBe` RequiresNoMagic
+
+getRNM
+    :: Genesis.Config
+    -> WalletConfiguration
+    -> TxpConfiguration
+    -> NtpConfiguration
+    -> IO RequiresNetworkMagic
+getRNM cfg _ _ _ =
+    pure $ getRequiresNetworkMagic $ Genesis.configProtocolMagic cfg
+
 spec :: Spec
 spec = describe "Pos.Launcher.Configuration" $ do
     describe "withConfigurations" $ do
-        it "should parse `lib/configuration.yaml` file" $ do
-            liftIO setupTestLogging
+        it ("should parse `" <> configFilePath <> "` file") $ do
+            liftIO $ setupTestLogging
             startTime <- Timestamp . round . (* 1000000) <$> liftIO getPOSIXTime
             let cfo = defaultConfigurationOptions
-                        { cfoFilePath = "./configuration.yaml"
+                        { cfoFilePath = configFilePath
```

Comment:
The `"./"` prefix might be important in some contexts, I would keep it in there.

Reply:
@Jimbo4350 asked about this - I'm not sure that presence/absence of a `"./` is an issue here, but given that the code has been working with the prefix, I'd hesitate to casually drop it. I'm not sure that all architectures treat said prefixes similarly.

Reply:
My suggestion is:
```
{ cfoFilePath = "./" <> configFilePath
```